packages/client/package.json

```diff
@@ -87,7 +87,10 @@
   },
   "dependencies": {
     "@grpc/grpc-js": "^1.8.16",
+    "@protobuf-ts/grpc-transport": "^2.9.0",
+    "@protobuf-ts/grpcweb-transport": "^2.9.0",
     "@protobuf-ts/runtime-rpc": "^2.9.0",
-    "apache-arrow": "^12.0.1"
+    "apache-arrow": "^12.0.1",
+    "grpc-web": "^1.4.2"
```

Comment:
Is this package necessary also for "server" client? Can we move it into `packages/client-browser/package.json`?